frontend/test/playwright/e2e/report-media.spec.ts

```diff
@@ -23,8 +25,19 @@ test.describe.configure({ mode: "parallel" })
 
 const reportingEndpoint = "**/report/"
 
+const reportButtonLabel = t("mediaDetails.contentReport.long")
+
 export const openReportModal = (page: Page) =>
-  page.click('text="Report this content"')
+  page.getByRole("button", { name: reportButtonLabel }).click()
+
+const checkOption = async (page: Page, option: ReportReason) => {
+  const name = t(`mediaDetails.contentReport.form.${option}.option`)
+  if (!(await page.getByRole("radio", { name }).isChecked())) {
+    // `.select()` doesn't work for radio buttons because Playwright
+    // incorrectly states that the element is covered by the icon SVG.
```

Comment:
If there is a bug report or something to link to to explain why Playwright _incorrectly_ reports this is the case, it would be helpful to document it here. Otherwise, it seems to easy to write Playwright's error off as "incorrect" when there may actually be an underlying issue we can solve. IIRC there is a bug report about something related to SVGs and click events.

I vaguely remember something like this being the case when working on https://github.com/WordPress/openverse/pull/4088, and it ended up being a bug in our pointer-events configuration for the checkboxes, leading to this change: https://github.com/WordPress/openverse/pull/4088/files#diff-9cab0d134035b3db0d42b453e18b3fff640f27e4155601385449cbe17b7788f8R50

When I was working on that, I remember coming across a Playwright bug report that seemed to indicate there was a general problem with Playwright's "click" and SVGs and that the issue seemed to imply it was just broken, rather than there being a real fix for it (in this case, passing through pointer events). In the end there was a fix in our code that did actually resolve a real (potential) problem. Apparently click events "happened" to work, but didn't technically always appear like they would work (which could affect accessibility tools maybe?).

If either is the case here, if it is a Playwright bug that we cannot do anything to fix, or if there is something we can change about the radio buttons to fix it, like there was for the checkboxes, then we should include that here explicitly.

Reply:
I thought there was a bug because I was adding `pointer-events-none` to the `VIcon.vue` component, and it was still getting the click event. Only to realize that the element in the report component is `VSvg`! 🤦 

Added `pointer-events-none` to `VRadiomark` and removed the comment [4cf211e](https://github.com/WordPress/openverse/pull/4804/commits/4cf211e2de9a1f910feae1274987e8ca9d8ad33a)